### PR TITLE
JAMES-2954 Correct error root cause in WebAdmin's addresses groups error messages

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/GroupsRoutesTest.java [hunk 1]
```diff
@@ -587,7 +587,7 @@ void deleteMalformedAddressShouldReturnBadRequest() {
             assertThat(errors)
                 .containsEntry("statusCode", HttpStatus.BAD_REQUEST_400)
                 .containsEntry("type", "InvalidArgument")
-                .containsEntry("message", "The group is not an email address")
+                .containsEntry("message", "The member is not an email address")
```
Comment (chibenwa) [inline server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/GroupsRoutesTest.java]: s/member/group member/ ?

File: server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java [hunk 2]
```diff
@@ -136,17 +137,17 @@ public void define(Service service) {
     })
     @ApiResponses(value = {
         @ApiResponse(code = HttpStatus.NO_CONTENT_204, message = "OK", response = List.class),
-        @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = GROUP_ADDRESS + " or group structure format is not valid"),
+        @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = GROUP_ADDRESS + "," + GROUP_ADDRESS + " or " + USER_ADDRESS + " format is not valid"),
```
Comment (Arsnael) [inline server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java]: Got twice `GROUP_ADDRESS` in the message?

File: server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java [hunk 3]
```diff
@@ -187,13 +188,13 @@ private void ensureNotShadowingAnotherAddress(MailAddress groupAddress) throws U
     @ApiResponses(value = {
         @ApiResponse(code = HttpStatus.OK_200, message = "OK", response = List.class),
         @ApiResponse(code = HttpStatus.BAD_REQUEST_400,
-            message = GROUP_ADDRESS + " or group structure format is not valid"),
+            message = GROUP_ADDRESS + "," + GROUP_ADDRESS + " or " + USER_ADDRESS + " format is not valid"),
```
Comment (Arsnael) [inline server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java]: Idem

**Comment by Arsnael** [comment]
CI restart...

test this please

**Comment by chibenwa** [comment]
test this please

File: server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java [hunk 4]
```diff
@@ -188,7 +188,7 @@ private void ensureNotShadowingAnotherAddress(MailAddress groupAddress) throws U
     @ApiResponses(value = {
         @ApiResponse(code = HttpStatus.OK_200, message = "OK", response = List.class),
         @ApiResponse(code = HttpStatus.BAD_REQUEST_400,
-            message = GROUP_ADDRESS + "," + GROUP_ADDRESS + " or " + USER_ADDRESS + " format is not valid"),
+            message = GROUP_ADDRESS + "," + GROUP_ADDRESS + " format is not valid"),
```
Comment (Arsnael) [inline server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java]: Didn't remove the right one here :)

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**